files/icons/material-nf.toml

```diff
@@ -21,91 +21,95 @@ backlight = [
     "\ue3c8", # nf-weather-moon_alt_waxing_crescent_1
     "\ue39b", # nf-weather-moon_full
 ]
-bat_charging = "\uf583" # nf-mdi-battery_charging
-bat_not_available = "\uf590" # nf-mdi-battery_unknown
+bat_charging = "\U000f0084" # nf-md-battery_charging
+bat_not_available = "\U000f0091" # nf-md-battery_unknown
 bat = [
-    "\uf579", # nf-mdi-battery_10
-    "\uf57a", # nf-mdi-battery_20
-    "\uf57b", # nf-mdi-battery_30
-    "\uf57c", # nf-mdi-battery_40
-    "\uf57d", # nf-mdi-battery_50
-    "\uf57e", # nf-mdi-battery_60
-    "\uf57f", # nf-mdi-battery_70
-    "\uf580", # nf-mdi-battery_80
-    "\uf581", # nf-mdi-battery_90
-    "\uf578", # nf-mdi-battery
+    "\U000f007a", # nf-md-battery_10
+    "\U000f007b", # nf-md-battery_20
+    "\U000f007c", # nf-md-battery_30
+    "\U000f007d", # nf-md-battery_40
+    "\U000f007e", # nf-md-battery_50
+    "\U000f007f", # nf-md-battery_60
+    "\U000f0080", # nf-md-battery_70
+    "\U000f0081", # nf-md-battery_80
+    "\U000f0082", # nf-md-battery_90
+    "\U000f0079", # nf-md-battery
 ]
-bell = "\uf599" # nf-mdi-bell
-bell-slash = "\uf59a" # nf-mdi-bell_off
-bluetooth = "\uf5ae" # nf-mdi-bluetooth
-calendar = "\uf5ec" # nf-mdi-calendar
-cogs = "\uf992" # nf-mdi-settings
+bell = "\U000f009c" # nf-md-bell_outline
+bell-slash = "\U000f009b" # nf-md-bell_off
```

Comment:
Why outline? And why only for `bell` but not `bell-slash`?

Reply:
It matched what had been done in fa. I think that it's easier to tell the difference when one is the outline and one is filled. (I did the same with the toggles)

Reply:
Now these two icons are not perfectly aligned: 

https://user-images.githubusercontent.com/34583604/219849036-6fb130df-5d96-4dcf-a68d-7e9c63696bba.mp4

Reply:
(does this mp4 work for anyone?)

Reply:
> (does this mp4 work for anyone?)

![image](https://user-images.githubusercontent.com/20397027/219849497-4d9f307c-0e5b-41c6-a9c5-d1e65adc9a8f.png)

Reply:
It works in `mpv` :(

Reply:
That's interesting, it doesn't move for me


https://user-images.githubusercontent.com/1033745/219849545-422d04b5-7354-4f3a-aa19-27dfc8fda973.mp4

Reply:
![image](https://user-images.githubusercontent.com/34583604/219849605-4ba82a5b-1697-44e3-8f91-6e4090f6eed8.png)

![image](https://user-images.githubusercontent.com/34583604/219849608-4fb16e63-0163-410a-beca-d3c3081596f1.png)

Reply:
It shifts, but less... Anyway, this is not a big deal, just a bit weird.